translator/src/main/resources/com/google/devtools/j2objc/JRE.mappings

```diff
@@ -98,12 +104,18 @@ java.lang.Class.isInterface()Z = isInterface
 java.lang.Class.isPrimitive()Z = isPrimitive
 java.lang.Class.newInstance()Ljava/lang/Object; = newInstance
 
+# java.lang.Cloneable
+java.lang.Cloneable.clone()Ljava/lang/Object; = javaClone
+
+# java.lang.Comparable
+java.lang.Comparable.compareTo(Ljava/lang/Object;)I = javaCompareTo:
+
 # java.lang.String
 java.lang.String.charAt(I)C = charAtWithInt:
 java.lang.String.codePointAt(I)I = codePointAt:
 java.lang.String.codePointBefore(I)I = codePointBefore:
 java.lang.String.codePointCount(II)I = codePointCount:endIndex:
-java.lang.String.compareTo(Ljava/lang/String;)I = compareToWithId:
+java.lang.String.compareTo(Ljava/lang/String;)I = javaCompareTo:
```

Comment:
I had just updated the existing entry without thinking about it, but looks like you're right. Removed.